tools/ovc/openvino/tools/ovc/__init__.py

```diff
@@ -18,3 +19,8 @@
 if is_optimum() and optimum_version is not None:
     telemetry = init_ovc_telemetry("Optimum Intel", optimum_version)
     telemetry.send_event("optimum", "import", "import_from_optimum,ov_version:{}".format(get_rt_version()))
+
+if is_torch() and 'torch' in sys.modules:
+    torch_version = importlib_metadata.version("torch")
+    telemetry = init_ovc_telemetry("PyTorch", torch_version)
+    telemetry.send_event("torch", "import", "Import from torch.compile")
```

Comment:
Do we want to include ov version like is done with optimum?

Reply:
Done.